circle.yml

```diff
@@ -0,0 +1,19 @@
+machine:
+    timezone:
+        Asia/Taipei
+
+    environment:
+        CIRCLE_ENV: test
+
+    post:
+        - pyenv global 2.7.10 3.5.0
```

Comment:
Are you sure CircleCI really use these version? 
Or it just use the system default. 
Maybe you can use `machine.python.version: 2.7.12` at this moment because we are not ready for testing multiple python versions.